src/renderable/animationsheet.js

```diff
@@ -19,11 +19,12 @@
      * @param {Number} [settings.framewidth] Width of a single frame within the spritesheet
      * @param {Number} [settings.frameheight] Height of a single frame within the spritesheet
```

Comment:
That's a good point. The debug panel is something I've struggled with in the past, as well. I would be cautious about changing the geometry methods (`left` getter et al.) to require information from the parent, like its anchorPoint. We don't have this requirement now, which allows things like inserting a sprite directly into the scene graph without a parent entity.

*on the other hand* the entity draw method just needs to translate to its own anchorPoint before drawing the child sprite. Then the sprite translates to its anchorPoint as normal. This translation step is clearly missing from the debug panel: https://github.com/melonjs/melonJS/blob/64eeb9996505766a8d3da60339aa5e45d7c18b13/plugins/debug/debugPanel.js#L200